api/graphql/users/user_filtersets.py

```diff
@@ -0,0 +1,10 @@
+import django_filters
+
+
+class UsersFilterSet(django_filters.FilterSet):
+    def filter_queryset(self, queryset):
```

Comment:
Would be in line with existing implementations to move this logic into schema.py's `UsersFilter` class in the method `resolve_queryset`

Reply:
I managed to do this, but when I removed `UsersFilterSet` class from the code, I started to get errors about missing `Meta.fields` from `UsersFilterSet` class. I didn't really understand where it comes from. I'm assuming it is raising from Django base class or something like that because I got this error even when I removed `UsersFilterSet` class.

I solved the issue by leaving the class but making it empty with `pass`. Let me know if you know a better way to handle this ☺️

Reply:
perhaps we need some follow up ticket to enhance this to something like `user is in at least one of these roles` type of check but that needs refinement so this is fine for now.

Reply:
Managed to get this fixed with your help! 🥳

Reply:
Yup, slacked.